vllm/model_executor/layers/logits_processor.py

```diff
@@ -78,16 +77,8 @@ def forward(
 
         return logits
 
-    def _get_logits(
-        self,
-        hidden_states: torch.Tensor,
-        lm_head: VocabParallelEmbedding,
-        embedding_bias: Optional[torch.Tensor],
-    ) -> Optional[torch.Tensor]:
-        # Get the logits for the next tokens.
-        logits = lm_head.linear_method.apply(lm_head,
-                                             hidden_states,
-                                             bias=embedding_bias)
+    def _gather_logits(self, logits: torch.Tensor) -> torch.Tensor:
```

Comment:
Refactor : introduce `_gather_logits()` that  `LogitsProcessorWithLoRA` also uses.